lib/view/help/help_frame.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:hakondate/constant/app_color.dart';
+import 'package:hakondate/state/help/help_state.dart';
+import 'package:hakondate/view/component/label/descpription_text.dart';
+import 'package:hakondate/view_model/single_page/help/help_view_model.dart';
+
+class HelpFrame extends StatelessWidget {
+  const HelpFrame({
+    required this.label,
+    required this.content,
+    super.key,
+  });
+
+  factory HelpFrame.baseNut() => HelpFrame(
+        label: '基準値について',
```

Comment:
インデントが二つ分入ってる。
全体的に上げる。
```suggestion
  factory HelpFrame.baseNut() => HelpFrame(
    label: '基準値について',
```

Reply:
何ヶ所か崩れているので全体的に見直す。